cmd/bosun/conf/conf.go

```diff
@@ -297,11 +297,14 @@ type Notification struct {
 	Timeout      time.Duration
 	ContentType  string
 	RunOnActions bool
+	UseBody      bool
```

Comment:
I think @mjibson means to remove this altogether, and make no conf changes. Just change the data object passed into the body template here:https://github.com/bosun-monitor/bosun/blob/master/cmd/bosun/conf/notify.go#L49 to have both the subject and body available. 

This will be a breaking change, but I think it will be more flexible that way.